_posts/2020-11-11-cockpit-232.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Cockpit 232
+author: kkoukiou
+date: 2020-11-11
+tags: cockpit linux
+slug: cockpit-232
+category: release
+summary: Cockpit with Dashboard and Machines improvements
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly. Here are the release notes from Cockpit version 232.
+
+### Improved editing in the host switcher
```

Comment:
### Improved host name editing

Reply:
I don't think it's about hostname editting. Hmm